CHANGELOG.rst

```diff
@@ -9,6 +9,8 @@ Changelog
 
 .. note:: This version is not yet released and is under active development.
 
+* Python 2.6 support is deprecated, and will be removed in the next release of
```

Comment:
Should we bold this?

Reply:
Any particular words you'd like bold, or want a `**DEPRECATION**` or `**BACKWARDS INCOMPATIBLE**` or something?

Reply:
**IMPENDING DOOM WARNING** 😀 

**FINAL DEPRECATION** might be okay. "Deprecation alone" seems insufficient? Or **SUPPORT CHANGE**